conf/application.conf

```diff
@@ -113,11 +113,9 @@ auth.google.domainWhitelist = ["lucidchart.com"]
 # You can configure a non-default credentials file to load AWS credentials from
 # aws.credentialsFile = /etc/aws/dev-credentials
 amazon.s3.enabled = true
-amazon.s3.bucketCloudfrontMap {
-	"dev-cashy" = {
-		cloudfront = "http://d2zt0093sgpvii.cloudfront.net/"
-	}
-}
+amazon.s3.bucketCloudfrontMap = {}
+# Example:
+# amazon.dev-cashy.bucketCloudfrontMap.dev-cashy.cloudfront =  "http://d2zt0093sgpvii.cloudfront.net/"
```

Comment:
**consider:** leaving a comment explaining what this used to be/was intended for or why disabled (e.g., it was never utilized/working)